forge-gui/res/cardsfolder/upcoming/three_bowls_of_porridge.txt

```diff
@@ -0,0 +1,10 @@
+Name:Three Bowls of Porridge
+ManaCost:2
+Types:Artifact Food
+A:AB$ Charm | Cost$ 2 T | Choices$ DBDamage,DBTap,DBSac | ChoiceRestriction$ ThisGame | CharmNum$ 1
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Creature | NumDmg$ 2 | StackDescription$ CARDNAME deals 2 damage to target creature. | SpellDescription$ CARDNAME deals 2 damage to target creature.
```

Comment:
what's the point of these duplicate StackDesc?

Reply:
otherwise it would put the whole spelldescription on the stack. Don't know if there's an easier way to do this

Reply:
got a screenshot?
looks fine here without 😕

Reply:
ah nvm. You're right